contracts/src/Teleporter/TeleporterMessenger.sol

```diff
@@ -33,46 +37,47 @@ contract TeleporterMessenger is ITeleporterMessenger, ReentrancyGuards {
+
     // Tracks the latest message ID used for a given destination chain.
-    // Key is the destination chain ID, and the value is the last message ID used for that chain.
+    // Key is the destination blockchain ID, and the value is the last message ID used for that chain.
     // Note that the first message ID used for each chain will be 1 (not 0).
-    mapping(bytes32 => uint256) public latestMessageIDs;
-
-    // Tracks the outstanding receipts to send back to a given chain in subsequent messages sent to it.
-    // Key is the other chain ID, and the value is a queue of pending receipts for messages
-    // we have received from that chain.
-    mapping(bytes32 => ReceiptQueue.TeleporterMessageReceiptQueue)
-        public outstandingReceipts;
-
-    // Tracks the message hash and fee information for each message sent that we have not yet received
-    // a receipt for. The messages are tracked per chain and keyed by message ID.
-    // The first key is the chain ID, the second key is the message ID, and the value is the info
+    mapping(bytes32 destinationChainID => uint256 messageID)
+        public latestMessageIDs;
+
+    // Tracks the outstanding receipts to send back to a given chain in subsequent messages sent to that chain.
+    // Key is the blockchain ID of the other chain, and the value is a queue of pending receipts for messages
+    // received from that chain.
+    mapping(bytes32 sourceChainID => ReceiptQueue.TeleporterMessageReceiptQueue receiptQueue)
+        public receiptQueues;
+
+    // Tracks the message hash and fee information for each message sent that has yet to be acknowledged
+    // with a receipt. The messages are tracked per chain and keyed by message ID.
+    // The first key is the blockchain ID of the destination chain, the second key is the message ID, and the value is the info
     // for the uniquely identified message.
-    mapping(bytes32 => mapping(uint256 => SentMessageInfo))
+    mapping(bytes32 destinationChainID => mapping(uint256 messageID => SentMessageInfo messageInfo))
         public sentMessageInfo;
 
-    // Tracks the relayer reward address for each message delivered from a given chain.
-    // Note that these values are also used to determine if a given message has been delivered or not.
-    // The first key is the chain ID, the second key is the message ID, and the value is the reward address
-    // provided by the deliverer of the uniquely identified message.
-    mapping(bytes32 => mapping(uint256 => address))
-        public relayerRewardAddresses;
-
     // Tracks the hash of messages that have been received but whose execution has never succeeded.
     // Enables retrying of failed messages with higher gas limits. Message execution is guaranteed to
-    // succeed at most once.  The first key is the chain ID, the second key is the message ID, and
+    // succeed at most once. The first key is the blockchain ID, the second key is the message ID, and
```

Comment:
```suggestion
    // succeed at most once. The first key is the blockchain ID of the sending chain, the second key is the message ID, and
```

Reply:
The first time I suggested a change on this line, I had inadvertently suggested a change that was already included.